src/sidebar/components/ModerationBanner.js

```diff
@@ -30,14 +30,22 @@ function ModerationBanner({ annotation, api, toastMessenger }) {
   const isHiddenOrFlagged =
     flagCount !== null && (flagCount > 0 || annotation.hidden);
 
+  if (!isHiddenOrFlagged) {
+    return null;
+  }
+
+  // In order to have been hidden or flagged, this annotation must have been
+  // saved.
```

Comment:
"Saved" as proven by a present value for `id`?

Reply:
Yes, exactly.